src/scripts/theme.js

```diff
@@ -31,4 +32,9 @@ $(document).ready(function() {
   // Wrap videos in div to force responsive layout.
   slate.rte.wrapTable();
   slate.rte.iframeReset();
+
+  // Add a message to the cart if cookies are disabled.
```

Comment:
This function won't directly display the message, that's taken care of by CSS.  Suggested reword:

> Apply a specific class to the html element for browser support of cookies.